kubectl-fdb/cmd/cordon_test.go

```diff
@@ -32,110 +32,190 @@ import (
 					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					clusterName:  clusterName,
+					clusterLabel: "",
 				}),
 			Entry("Cordon no node nodes without exclusion",
 				testCase{
 					nodes:                     []string{""},
 					WithExclusion:             false,
 					ExpectedInstancesToRemove: []string{},
 					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					clusterName:  clusterName,
+					clusterLabel: "",
 				}),
 			Entry("Cordon all nodes with exclusion",
 				testCase{
 					nodes:                     []string{"node-1", "node-2"},
 					WithExclusion:             true,
 					ExpectedInstancesToRemove: []string{"instance-1", "instance-2"},
 					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					clusterLabel: fdbv1beta2.FDBClusterLabel,
+					clusterName:  clusterName,
 				}),
 			Entry("Cordon all nodes without exclusion",
 				testCase{
 					nodes:                     []string{"node-1", "node-2"},
 					WithExclusion:             false,
 					ExpectedInstancesToRemove: []string{},
 					ExpectedInstancesToRemoveWithoutExclusion: []string{"instance-1", "instance-2"},
+					clusterName:  clusterName,
+					clusterLabel: fdbv1beta2.FDBClusterLabel,
+				}),
+			Entry("Cordon node from second cluster without exclusion",
+				testCase{
+					nodes:                     []string{"node-1"},
+					WithExclusion:             true,
+					ExpectedInstancesToRemove: []string{"instance-3"},
+					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					clusterName:  secondClusterName,
+					clusterLabel: fdbv1beta2.FDBClusterLabel,
+				}),
+			Entry("Cordon node from all clusters with exclusion",
+				testCase{
+					nodes:                     []string{"node-1"},
+					WithExclusion:             true,
+					ExpectedInstancesToRemove: []string{"instance-1", "instance-3"},
+					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					clusterName:  "",
+					clusterLabel: fdbv1beta2.FDBClusterLabel,
+				}),
+			Entry("Cordon all node from all clusters without exclusion",
+				testCase{
+					nodes:                     []string{"node-1", "node-2"},
+					WithExclusion:             false,
+					ExpectedInstancesToRemove: []string{},
+					ExpectedInstancesToRemoveWithoutExclusion: []string{"instance-1", "instance-2", "instance-3", "instance-4"},
+					clusterName:  "",
+					clusterLabel: fdbv1beta2.FDBClusterLabel,
+				}),
+			Entry("Cordon no node nodes without exclusion with custom label",
+				testCase{
+					nodes:                     []string{""},
+					WithExclusion:             false,
+					ExpectedInstancesToRemove: []string{},
+					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					clusterName:  "",
+					clusterLabel: fdbv1beta2.FDBClusterLabel,
 				}),
 		)
 	})
 })
+
+func createPods(inputClusterName string, inputNamespace string, id int) error {
+	pods := []corev1.Pod{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      fmt.Sprintf("instance-%d", id),
```

Comment:
Is there a reason we have to use the ID? It's probably better to prefix the Pod name with the cluster name, that way it's pretty obvious which Pods are associated with which cluster.